orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -3045,7 +3045,7 @@ void ConvTransposeGradientCheckerTest(std::vector<std::unique_ptr<IExecutionProv
   GradientChecker<float, float, float> gradient_checker;
   OpDef op_def{"ConvTranspose"};
 
-  float error_tolerance = 1e-1f;
+  float error_tolerance = 3e-1f;
```

Comment:
A quick question: we defined a relatively higher tolerance for `ConvTransposeGradientCheckerTest` and `ConvGradientCheckerTest`, is there any explanation for it, I mean, is it expected for the small-sized test inputs?

Reply:
I don't have a good answer for this question. @er3x3 do you have an answer for this? I could only speculate. I can probably delve deeper and find an answer to this, but I don't know as of now.

Reply:
cuDNN is non-deterministic, while the expected values are computed on CPU.